index.js

```diff
@@ -72,14 +95,31 @@ async function main(adapterAddress, startBlock = 0, endBlock = Infinity) {
 
   const totalScore = Object.values(scores).reduce((acc, { score }) => acc.plus(score), new Decimal(0));
 
+  const totalAvailableMorpho = dryRun
+    ? new Decimal(1337).times("1e18")
+    : new Decimal(await morphoToken.balanceOf(adapterAddress).then((b) => b.toString()));
+
   const rights = {};
   for (const [user, { score }] of Object.entries(scores)) {
-    rights[user] = score.div(totalScore).toNumber();
+    const _score = score.div(totalScore);
+    if (_score.toNumber() > 0.001)
+      rights[user] = _score.times(totalAvailableMorpho).toDecimalPlaces(0).toString();
```

Comment:
Why do we round here? IIUC, this means that there would be some MORPHO dust left since we need to pass the amount to claim as well to the Rewards Distributor.

Reply:
Well we have to round one way or the other since we need to get things to 18 decimal point uints! If there is a tiny amount of dust (~ 1e-16), that's acceptable, IMO

Reply:
Ah, I see. 👍🏻